cli/core/hack/rpm/build_package.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+if [ $(uname) != "Linux" ]; then
+   echo "This script must be run on a Linux system"
+   exit 1
+fi
+
+# Use DNF and if it is not installed fallback to YUM
+DNF=$(command -v dnf || command -v yum || true)
+if [ -z "$DNF" ]; then
+   echo "This script requires the presence of either DNF or YUM package manager"
+   exit 1
+fi
+
+# VERSION should be set when calling this script
+if [ -z "${VERSION}" ]; then
+   echo "\$VERSION must be set before calling this script"
+   exit 1
+fi
+exit
```

Comment:
Well, that is a mistake!  Left over from a test I was running.  Sorry about that.